### DISCUSSION TITLE
Projects fail to open after download from locally hosted QFieldCloud

### DISCUSSION THREAD

**Discussion by BizTechnology** [opengisch/QField#5715] [Bugs]
![qfield1](https://github.com/user-attachments/assets/fd6378d0-4ac1-414d-aefb-4fe7e2342337)

I'm currently facing some issues with our locally deployed QFieldCloud instance and could really use some help troubleshooting.

I've deployed QFieldCloud on a local server and am able to upload projects from QGIS to the locally deployed QFieldCloud instance successfully. However, QGIS has successfully uploaded the project to the QFieldCloud but when downloading it using QField, the following errors are displayed in the QField.

The errors I'm seeing in the QField are:

Project file read error in file c:/users/bizit/appdata/roaming/opengis.ch/qfield/cloud_projects/admin/5a536cbb-a423-4835-845f-85594bcdbe34/gis4_cloud_qfield.qgs: premature end of document at line 1, column 0
Error opening zip archive: 'c:/users/bizit/appdata/roaming/opengis.ch/qfield/cloud_projects/admin/5a536cbb-a423-4835-845f-85594bcdbe34/gis4_cloud_qfield_attachments.zip' (error code: 19) 
Failed to remove temporary file gis4_cloud_qfield_attachments.zip
Failed to remove temporary file gis4_cloud_qfield.qgs
Failed to remove temporary file data.gpkg

And when I try to download individual file from the QFieldCloud projects overview in QGIS, the downloaded file will become 0.00KB (zero kilo bites) and no xml code in the file. Kindly checkout the snapshot.
Has anyone else seen these errors when downloading projects from a locally hosted QFieldCloud instance? Any suggestions on how to troubleshoot or resolve the errors would be appreciated.

Any help pls. Or can anyone send us a proven working procedure to follow to install QFieldCloud locally that smoothly works with the QField. I've follow the GitHub steps but QField is facing problem to download and display the upload projects from locally deployed QFieldCloud.

Thanks.

**Comment by julj** [comment]
I have a similar issue: the app seems to manage to download the project, but it when opening the project I get no layers. In the log I have the same errors:

```
Project file read error in file /[data/user/0/ch.opengis.qfield/files/cloud_projects/jricard/855d35b5-7d0d-4bb2-8497-4fa71c859ea5/trees_cloud_qfield.qgs](http://data/user/0/ch.opengis.qfield/files/cloud_projects/jricard/855d35b5-7d0d-4bb2-8497-4fa71c859ea5/trees_cloud_qfield.qgs): Premature end of document. at line 1 column 0

Error opening zip archive: '/[data/user/0/ch.opengis.qfield/files/cloud_projects/jricard/855d35b5-7d0d-4bb2-8497-4fa71c859ea5/trees_cloud_qfield_attachments.zip'](http://data/user/0/ch.opengis.qfield/files/cloud_projects/jricard/855d35b5-7d0d-4bb2-8497-4fa71c859ea5/trees_cloud_qfield_attachments.zip') (Error code: 19)
```

**Comment by suricactus** [comment]
Duplicate of https://github.com/opengisch/QFieldCloud/issues/1044
